docs/docs/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Setting Up and Running Jenkins
+---
+
+## {{page.title}}
```

Comment:
Oh, I guess this only works on the Jekyll site, not when looking at the Markdown directly in the GitHub UI.